distribution/openrct2.d.ts

```diff
@@ -307,46 +475,71 @@ declare global {
         "research" |
         "interest";
 
+    /** Arugments for the GameActionEvent hook callback. */
     interface GameActionEventArgs {
+        /** The number of the player who executed the action. TODO always 0 in single player? */
         readonly player: number;
+        /** The event type. */
         readonly type: string;
+        /** TODO True if the event only occurs in the client, false if needs to be synchronized to the network? */
         readonly isClientOnly: boolean;
+        /** TODO */
         readonly args: object;
+        /** The result of the action. */
         result: GameActionResult;
     }
 
+    /** The result of the game action. */
     interface GameActionResult {
+        /** TODO? */
         error?: number;
+        /** TODO? */
         errorTitle?: string;
+        /** The error message which is displayed in a pop-up, if specified. */
         errorMessage?: string;
+        /** The position of the error message, if any. TODO position of the error or just the position of the action? */
```

Comment:
It is the centre of the action (i.e. if the land modify action was 4x4 it would be the centre of the 4x4) or if the action does not make sense to have a position this will be null (This at least is how it works in C++ @IntelOrca  will need to confirm that is still how it works in plugins)